tensorflow/contrib/distributions/python/kernel_tests/student_t_test.py

```diff
@@ -77,6 +77,32 @@ def testStudentLogPDFMultidimensional(self):
       self.assertAllClose(expected_pdf, pdf_values)
       self.assertAllClose(np.exp(expected_log_pdf), pdf_values)
 
+  def testStudentCDFAndLogCDF(self):
+    with self.test_session():
+      batch_size = 6
+      df = tf.constant([3.] * batch_size)
+      mu = tf.constant([7.] * batch_size)
+      sigma = tf.constant([8.] * batch_size)
+      df_v = 3.
+      mu_v = 7.
+      sigma_v = 8.
+      t = np.array([-2.5, 2.5, 8., 0., -1., 2.], dtype=np.float32)
+      student = tf.contrib.distributions.StudentT(df, mu=mu, sigma=sigma)
+
+      log_cdf = student.log_cdf(t)
+      self.assertEquals(log_cdf.get_shape(), (6,))
+      log_cdf_values = log_cdf.eval()
+      cdf = student.cdf(t)
+      self.assertEquals(cdf.get_shape(), (6,))
+      cdf_values = cdf.eval()
+
+      expected_log_cdf = stats.t.logcdf(t, df_v, loc=mu_v, scale=sigma_v)
+      expected_cdf = stats.t.cdf(t, df_v, loc=mu_v, scale=sigma_v)
+      self.assertAllClose(expected_log_cdf, log_cdf_values)
```

Comment:
I've just discovered that this test is riddled with flaky unit-tests; this may be partly to blame for your problems.  Ill try to get the fix submitted ASAP.